docs/config/lua/pane/paste.md

```diff
@@ -2,11 +2,11 @@
 
 {{since('20201031-154415-9614e117')}}
 
-Sends the supplied `text` string to the input of the pane as if it
-were pasted from the clipboard, except that the clipboard is not involved.
-
-If the terminal attached to the pane is set to bracketed paste mode then
-the text will be sent as a bracketed paste.
-
-Otherwise the string will be streamed into the input in chunks of
-approximately 1KB each.
+!!! info "*Since: Version 20221119-145034-49b9839f*"
+    Alias of [`send_paste`](send_paste.md) for backwards compatibility with prior releases.
+
+{%
+    include-markdown "./send_paste.md"
+    start="<!--shared-paste-description-BEGIN-->"
+    end="<!--shared-paste-description-END-->"
+%}
```

Comment:
this is neat, but forms a DAG to the other page, and it is not obvious in that page why those blocks are present.

I think we should just remove this; the link to the page above is sufficient.
```suggestion
```